prime.py

```diff
@@ -0,0 +1,12 @@
+num=13
+count=2
+while count!=num:
+     if num%count==0:
+       break
+else:
+     count+=1
+
+if count==num:
+   print "Number",num,"is prime"
+else:
+  print "Number",num,"is not prime"
```

Comment:
Need to take input from the user here.

Also once we find that any number is divisible, need to set flag and break and outside the for loop just see if flag is set to check if number is prime